views/random-beer.hbs

```diff
@@ -0,0 +1,16 @@
+<div>
+    {{#each responseFromAPI}}
+    <div class="random-beer-layout">
+
+        {{> beerpartial this}}
+    </div>
+    <div class="random-beer-bonus-info">
+        Food pairing: {{this.food_pairing}}
```

Comment:
since food_pairing is an array, you could have also use a @each to list them separately with a ul/ol tag.